pages/papers/how-to-create-gems.mdx

```diff
@@ -1,39 +1,40 @@
 ---
-title: How to create a gems
+title: How to create gems
 lang: en
 description: We will learn to create our first gem.
 ---
 
-<div css={{ color: "red", fontStyle: "italic", fontSize: "2em" }}>
-  In Progress
-</div>
-
 **Tags:** _Ruby_, _Ruby Gems_
 
-**Audience Level:** _~~Beginer~~_ _Intermediate_
+**Audience Level:** _Beginer_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
-You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
+  You have 300 characters to sell your talk. This is known as the "elevator
+  pitch". Make it as exciting and enticing as possible.
 </div>
 
-¿Porque son importantes las gemas en ruby? ¿Porque debería construir una?. El éxito de una gema dependerá mucho de patrones de desarrollo y la aplicación de buenas practicas. Entonces, aprenderemos a como construir una gema y divertirnos en el proceso.
+Why are gems in ruby important? Why should I build one? The success of a gem will depend a lot on patterns of development and the application of good practices. Then, we will learn how to build a gem and have fun in the process.
 
 ### Description
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
-You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
+  You should make the description of your talk as compelling and exciting as
+  possible. Remember, you're selling both the organizers of the events to select
+  your talk, as well as trying to convince attendees your talk is the one they
+  should see.
 </div>
 
-Si quieres empezar a contribuir a una comunidad creo que el primer gran paso seria creando tu propia gema, independientemente si soluciona un problema especifico para un monton de personas, o si solo lo haces para pasarla bien y divertirte haciendolo, o simplemente por curiosidad te inicias con un genial "Hola mundo, vengo en son de paz", son muchos los motivos por los cuales uno puede animarse a construir una gema, lo ideal seria identificar eso que siempre repites en todos los proyectos que pasaron por tus manos, y automatizarlo, como? construyendo una gema e invocandolo en todos los proyectos que necesites.
+If you want to start contributing to a community I think the first big step would be creating your own gem, regardless of whether it solves a specific problem for a lot of people, or if you just do it to have fun and have fun doing it, or just out of curiosity you start with a great "Hello world, I come in peace", there are many reasons why one can be encouraged to build a gem, the ideal would be to identify what you always repeat in all the projects that passed through your hands, and automate it, building a gem and invoking it in all the projects you need.
```

Comment:
to a community, I think the first big step [...] or if you just do it to have a great time and fun doing it, or just out of curiosity, you start with [...];* there are many reasons [...]. The* ideal thing would be to identify [...] projects that went through you hands[...].